components/tor/tor_launcher_factory.cc

```diff
@@ -55,12 +56,14 @@ TorLauncherFactory::TorLauncherFactory()
           {base::ThreadPool(), base::MayBlock(),
```

Comment:
it seems this one is only used for getting tor log, so better to create it in place when needed

Reply:
fixed in 47093e7b5b5e8e93172696c68a52086fb3a1aaf6